spec/services/email_due_soon_and_overdue_defects_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe EmailDueSoonAndOverdueDefects do
+  let(:defects) { create_list(:property_defect, 3) }
+
+  describe '#call' do
+    it 'emails the team' do
+      team_message_delivery = instance_double(ActionMailer::MessageDelivery)
+      expect(DueSoonAndOverdueDefectsMailer).to receive(:notify)
```

Comment:
Same as the previous comment re: testing the structure vs testing the behaviour 👍